src/FixedDataTableBufferedRows.js

```diff
@@ -50,7 +50,6 @@ var FixedDataTableBufferedRows = React.createClass({
   },
 
   componentDidMount() {
-    setTimeout(this._updateBuffer, 1000);
```

Comment:
It might be related to how the test is creating this..can you debug the callstack to ComponentDidMount?